src/WebEid.Security/Util/X509CertificateExtensions.cs

```diff
@@ -80,6 +82,7 @@ public static X509Certificate2 ValidateIsSignedByTrustedCa(this X509Certificate2
                     UrlRetrievalTimeout = TimeSpan.Zero
                 }
             };
+
```

Comment:
I don't see where you check CA certificate expiry? I expected to see it below around line 117, something like this:

```c#
ValidateCertificateExpiry(chainElement.Certificate, DateTimeProvider.UtcNow, "Trusted CA");
return chainElement.Certificate;
```

Or is this somehow handled implicitly?

Reply:
Good catch. I will fix this promptly.